h/security/permissions.py

```diff
@@ -1,8 +1,6 @@
 class Permission:
     class Annotation:
-        ADMIN = "annotation:admin"  # Is this used anywhere?
         READ = "annotation:read"
-        WRITE = "annotation:write"  # Is this granted anywhere?
```

Comment:
This is just an error, where I thought a scope with the same string was a permission